lighthouse-cli/performance-experiment/report/scripts/perf-x.js

```diff
@@ -31,7 +31,8 @@ class ConfigPanel {
     this._messageField = this._configPanel.querySelector('.js-message');
     this._urlBlockingList = this._configPanel.querySelector('.js-url-blocking-patterns');
     this._urlBlockingStatus = {};
-    this._reportId = new URL(window.location).searchParams.get('id');
+    const match = location.search.match(/[?&]id=([^&]*)/);
```

Comment:
thanks for your advice. Shall I move the dependencies from viewer to core? Or just add some dependencies to cli?